crates/parser/src/raydium/account_parser.rs

```diff
@@ -0,0 +1,118 @@
+    OperationState(OperationState),
+    ObservationState(ObservationState),
+    PersonalPositionState(PersonalPositionState),
+    PoolState(PoolState),
+    ProtocolPositionState(ProtocolPositionState),
+    TickArrayState(TickArrayState),
+    TickArrayBitmapExtension(TickArrayBitmapExtension),
+}
+impl RaydiumProgramState {
+    pub fn try_unpack(data_bytes: &[u8]) -> ParseResult<Self> {
+        let data_len = data_bytes.len();
+        let data_bytes = &data_bytes[ACC_DISCRIMINATOR_SIZE..];
+
+        match data_len {
+            AmmConfig::LEN => Ok(RaydiumProgramState::AmmConfig(AmmConfig::try_from_slice(
+                data_bytes,
+            )?)),
+            OperationState::LEN => Ok(RaydiumProgramState::OperationState(
+                OperationState::try_from_slice(data_bytes)?,
+            )),
+            ObservationState::LEN => Ok(RaydiumProgramState::ObservationState(
+                ObservationState::try_from_slice(data_bytes)?,
+            )),
+            PersonalPositionState::LEN => Ok(RaydiumProgramState::PersonalPositionState(
+                PersonalPositionState::try_from_slice(data_bytes)?,
+            )),
+            PoolState::LEN => Ok(RaydiumProgramState::PoolState(PoolState::try_from_slice(
+                data_bytes,
+            )?)),
+            ProtocolPositionState::LEN => Ok(RaydiumProgramState::ProtocolPositionState(
+                ProtocolPositionState::try_from_slice(data_bytes)?,
+            )),
+            TickArrayState::LEN => Ok(RaydiumProgramState::TickArrayState(
+                TickArrayState::try_from_slice(data_bytes)?,
+            )),
+            TickArrayBitmapExtension::LEN => Ok(RaydiumProgramState::TickArrayBitmapExtension(
+                TickArrayBitmapExtension::try_from_slice(data_bytes)?,
+            )),
+            _ => Err(ParseError::from("Invalid Account data length".to_owned())),
+        }
+    }
+}
+
+#[derive(Debug, Copy, Clone)]
+pub struct AccountParser;
+
+impl Parser for AccountParser {
+    type Input = yellowstone_vixen_core::AccountUpdate;
+    type Output = RaydiumProgramState;
+
+    fn id(&self) -> std::borrow::Cow<str> {
+        "yellowstone_vixen_parser::jup_programs::raydium::AccountParser".into()
+    }
+
+    fn prefilter(&self) -> Prefilter {
+        Prefilter::builder()
+            .account_owners([RADIUM_V3_PROGRAM_ID])
+            .build()
+            .unwrap()
+    }
+
+    async fn parse(
+        &self,
+        acct: &yellowstone_vixen_core::AccountUpdate,
+    ) -> ParseResult<Self::Output> {
+        let inner = acct.account.as_ref().ok_or(ProgramError::InvalidArgument)?;
+        RaydiumProgramState::try_unpack(&inner.data)
+    }
+}
+
+impl ProgramParser for AccountParser {
+    #[inline]
+    fn program_id(&self) -> yellowstone_vixen_core::Pubkey {
+        RADIUM_V3_PROGRAM_ID.to_bytes().into()
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use yellowstone_vixen_mock::{account_fixture, run_account_parse, FixtureData};
```

Comment:
```suggestion
    use yellowstone_vixen_mock::account_fixture;
```

We just need account_fixture now.